flow360/component/flow360_params/flow360_output.py

```diff
@@ -54,15 +54,17 @@ def _filter_fields(fields, literal_filter):
 class AnimationSettings(Flow360BaseModel):
     """:class:`AnimationSettings` class"""
 
-    frequency: Optional[Union[PositiveInt, Literal[-1]]] = pd.Field(alias="frequency")
+    frequency: Optional[Union[PositiveInt, Literal[-1]]] = pd.Field(
```

Comment:
why do we need Animated/Static here? Should we just assume AnimationSettings is present only when animation is set by user, otherwise object is not present hence static?